### ISSUE TITLE
Investigate modules for bitrot and cull ones that are non-functional

### ISSUE THREAD

**Issue by dbjergaard** [stumpwm/stumpwm-contrib#72]
This could be a potentially large project, but also fairly easy for someone new to Common Lisp.  We should all the modules and remove the non-functional ones.  Here's a check list:

- [ ] modules/media/amixer
- [ ] modules/media/aumix
- [ ] modules/minor-mode/mpd
- [ ] modules/minor-mode/notifications
- [ ] modules/modeline/battery
- [ ] modules/modeline/battery-portable
- [ ] modules/modeline/cpu
- [ ] modules/modeline/disk
- [ ] modules/modeline/hostname
- [ ] modules/modeline/maildir
- [ ] modules/modeline/mem
- [ ] modules/modeline/net
- [ ] modules/modeline/wifi
- [ ] modules/util/alert-me
- [ ] modules/util/app-menu
- [ ] modules/util/debian
- [ ] modules/util/ft2-fonts
- [ ] modules/util/globalwindows
- [ ] modules/util/kbd-layouts
- [ ] modules/util/logitech-g15-keysyms
- [ ] modules/util/numpad-layouts
- [ ] modules/util/passwd
- [ ] modules/util/perwindowlayout
- [ ] modules/util/productivity
- [ ] modules/util/qubes
- [ ] modules/util/sbclfix
- [ ] modules/util/screenshot
- [ ] modules/util/searchengines
- [ ] modules/util/stumpish
- [ ] modules/util/stumptray
- [ ] modules/util/surfraw
- [ ] modules/util/swm-emacs
- [ ] modules/util/ttf-fonts
- [ ] modules/util/undocumented
- [ ] modules/util/urgentwindows
- [ ] modules/util/windowtags
- [ ] modules/util/winner-mode

**Comment by lboc** [comment]
Hi @dbjergaard 

> but also fairly easy for someone new to Common Lisp.

This sounds like me. While the main stumpwm codebase is almost certainly beyond me at the moment, I'm willing to have a go at this if you like?

**Comment by dbjergaard** [comment]
Absolutely! Please do!  Your help is very much appreciated, even if you can check a few and report what you find.

**Comment by lboc** [comment]
OK, cool! I'll make a start sometime this week.

**Comment by lboc** [comment]
@dbjergaard Just sent my very first PR, so if I'm doing it wrong, please let me know!
I've checked the 'modeline' modules.
The 'battery' module doesn't work on my PC - looks like the paths and names used for battery info are wrong. 'battery-portable' seems to work fine though.

**Comment by lboc** [comment]
@dbjergaard Can't edit the checklist, but the modules/modeline ones are done.
Your PR comments noted, thanks. I'll see how things go with the other modules and try break things up differently next time.

**Comment by kriyative** [comment]
In addition to some of the modeline modules previously mentioned, I use `amixer` and `ttf-fonts` regularly and they are working well (at least on Ubuntu Linux 17.04).
